qt/python/mantidqtinterfaces/mantidqtinterfaces/TofConverter/convertUnits.py

```diff
@@ -11,6 +11,12 @@
 
 
 def doConversion(inputval, inOption, outOption, theta, flightpath):
+    if (
+        (inOption == MOMENTUM_TRANSFER or inOption == D_SPACING)
+        and (outOption == MOMENTUM_TRANSFER or outOption == D_SPACING)
+        and inOption != outOption
+    ):
```

Comment:
I think this is equivalent?

```suggestion
    if (inOption == MOMENTUM_TRANSFER and outOption == D_SPACING)
        or (inOption == D_SPACING and outOption == MOMENTUM_TRANSFER):
```